example/Pipfile

```diff
@@ -9,6 +9,11 @@ watchdog = "<0.10"
 [packages]
 gunicorn = "*"
 psycopg2-binary = ">=2.9, <3.0"
+
+# Uncomment the following line if you would like to run the dev without docker
+# django-polaris = {path = "..", editable = true}
```

Comment:
Is `..` correct? Is this path relative to the current working directory? For example, if I run from the `example` directory does the path need to be different than if I run from the `django-polaris` directory?

Reply:
Cannot find a way to inject a path with something like $HOME/django-polaris. 

The `pipenv install --dev` must be run from the example folder. This issue can be taken care of in the document.